### PR TITLE
desktopToDarwinBundle: fix 16x, 32x app icons

### LINKED ISSUE #358210: desktopToDarwinBundle: 16x and 32x app icons are broken
## Describe the bug

I noticed that some app bundles that are created through `desktopToDarwinBundle` have icons that appear corrupted when viewed in Finder. This seems to only apply to 16x16 and 32x32 icon sizes.

| 16x16 | 32x32 | 64x64 |
| ---- | ---- | ---- |
| image | image | image |

The affected apps in the screenshot are `dsview`, `ghidra` and `pulseview` (that version of `q2pro` is WIP and not present in nixpkgs yet, but it also suffers from the same problem).

App bundles created through different means don't seem to have this issue (example: `openboardview`, `kitty`, `1password`, `vscode`)

## Steps To Reproduce

1. Install `pulseview` (`systemPackages` or through `home-manager`)
2. Open nix application bundle directory in Finder
3. Set view icon size to 16x16 or 32x32

## Expected behavior

App icons should show up in Finder correctly 

## Metadata

```
 - system: `"aarch64-darwin"`
 - host os: `Darwin 24.1.0, macOS 15.1.1`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.18.8`
 - channels(carlossless): `"darwin, home-manager"`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
```

## Notify maintainers

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by carlossless**
Fixes https://github.com/NixOS/nixpkgs/issues/358210

Example using `pulseview` icon (`Applications/PulseView/Contents/Resources/pusleview.icns`) when viewed through [Icns Image Viewer](https://relikd.github.io/icnsutil/html/viewer.html)

| Before | After |
| ------ | ----- |
| relikd github io_icnsutil_html_viewer html (1) | relikd github io_icnsutil_html_viewer html |

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/development/python-modules/icnsutil/default.nix
Comment (carlossless) [inline pkgs/development/python-modules/icnsutil/default.nix:24]: Necessary for `icnsutil convert` to work

**Comment by carlossless** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 358247`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 5 packages built:</summary>
  <ul>
    <li>desktopToDarwinBundle</li>
    <li>python311Packages.icnsutil</li>
    <li>python311Packages.icnsutil.dist</li>
    <li>python312Packages.icnsutil</li>
    <li>python312Packages.icnsutil.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 5 packages built:</summary>
  <ul>
    <li>desktopToDarwinBundle</li>
    <li>python311Packages.icnsutil</li>
    <li>python311Packages.icnsutil.dist</li>
    <li>python312Packages.icnsutil</li>
    <li>python312Packages.icnsutil.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:fast_forward: 21 packages marked as broken and skipped:</summary>
  <ul>
    <li>ghidra</li>
    <li>ghidra-extensions.findcrypt</li>
    <li>ghidra-extensions.ghidra-delinker-extension</li>
    <li>ghidra-extensions.ghidra-golanganalyzerextension</li>
    <li>ghidra-extensions.gnudisassembler</li>
    <li>ghidra-extensions.lightkeeper</li>
    <li>ghidra-extensions.machinelearning</li>
    <li>ghidra-extensions.ret-sync</li>
    <li>ghidra-extensions.sleighdevtools</li>
    <li>gimpPlugins.exposureBlend</li>
    <li>gimpPlugins.resynthesizer</li>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>private-gpt</li>
    <li>private-gpt.dist</li>
    <li>python311Packages.private-gpt</li>
    <li>python311Packages.private-gpt.dist</li>
    <li>python312Packages.private-gpt</li>
    <li>python312Packages.private-gpt.dist</li>
    <li>rtfm</li>
    <li>tijolo</li>
  </ul>
</details>
<details>
  <summary>:x: 12 packages failed to build:</summary>
  <ul>
    <li>dune3d</li>
    <li>emojione</li>
    <li>gimpPlugins.bimp</li>
    <li>gimpPlugins.gap</li>
    <li>librum</li>
    <li>opcua-client-gui</li>
    <li>opcua-client-gui.dist</li>
    <li>python311Packages.pdf2docx</li>
    <li>python311Packages.pdf2docx.dist</li>
    <li>saga</li>
    <li>sioyek</li>
    <li>volantes-cursors</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 106 packages built:</summary>
  <ul>
    <li>apkleaks</li>
    <li>apkleaks.dist</li>
    <li>asciidoc-full</li>
    <li>asciidoc-full-with-plugins</li>
    <li>asciidoc-full-with-plugins.dist</li>
    <li>asciidoc-full.dist</li>
    <li>azpainter</li>
    <li>beamerpresenter</li>
    <li>browsr</li>
    <li>browsr.dist</li>
    <li>cadzinho</li>
    <li>dblatexFull</li>
    <li>ddd</li>
    <li>desktopToDarwinBundle</li>
    <li>diagrams-as-code</li>
    <li>diagrams-as-code.dist</li>
    <li>dsview</li>
    <li>element-desktop</li>
    <li>element-desktop-wayland</li>
    <li>fastqc</li>
    <li>gcs</li>
    <li>gimp</li>
    <li>gimp.dev</li>
    <li>gimpPlugins.farbfeld</li>
    <li>gimpPlugins.fourier</li>
    <li>gimpPlugins.gmic</li>
    <li>gimpPlugins.lightning</li>
    <li>gimpPlugins.lqrPlugin</li>
    <li>gimpPlugins.texturize</li>
    <li>gimpPlugins.waveletSharpen</li>
    <li>gpscorrelate</li>
    <li>gtkwave</li>
    <li>hackneyed</li>
    <li>hikounomizu</li>
    <li>inkscape</li>
    <li>inkscape-extensions.textext</li>
    <li>inkscape-extensions.textext.dist</li>
    <li>inkscape-with-extensions</li>
    <li>jadx</li>
    <li>k40-whisperer</li>
    <li>kakounePlugins.rep</li>
    <li>llpp</li>
    <li>logisim</li>
    <li>logisim-evolution</li>
    <li>lyra-cursors</li>
    <li>manga-cli</li>
    <li>mars-mips</li>
    <li>mcomix</li>
    <li>mcomix.dist</li>
    <li>meld</li>
    <li>mupdf</li>
    <li>mupdf.bin</li>
    <li>mupdf.dev</li>
    <li>mupdf.doc</li>
    <li>mupdf.man</li>
    <li>numix-cursor-theme</li>
    <li>pulseview</li>
    <li>python311Packages.diagrams</li>
    <li>python311Packages.diagrams.dist</li>
    <li>python311Packages.icnsutil</li>
    <li>python311Packages.icnsutil.dist</li>
    <li>python311Packages.llama-index</li>
    <li>python311Packages.llama-index-readers-file</li>
    <li>python311Packages.llama-index-readers-file.dist</li>
    <li>python311Packages.llama-index-readers-s3</li>
    <li>python311Packages.llama-index-readers-s3.dist</li>
    <li>python311Packages.llama-index.dist</li>
    <li>python311Packages.osc-diagram</li>
    <li>python311Packages.osc-diagram.dist</li>
    <li>python311Packages.pymupdf</li>
    <li>python311Packages.pymupdf.dist</li>
    <li>python311Packages.pytikz-allefeld</li>
    <li>python311Packages.pytikz-allefeld.dist</li>
    <li>python312Packages.diagrams</li>
    <li>python312Packages.diagrams.dist</li>
    <li>python312Packages.icnsutil</li>
    <li>python312Packages.icnsutil.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-readers-file</li>
    <li>python312Packages.llama-index-readers-file.dist</li>
    <li>python312Packages.llama-index-readers-s3</li>
    <li>python312Packages.llama-index-readers-s3.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.osc-diagram</li>
    <li>python312Packages.osc-diagram.dist</li>
    <li>python312Packages.pdf2docx</li>
    <li>python312Packages.pdf2docx.dist</li>
    <li>python312Packages.pymupdf</li>
    <li>python312Packages.pymupdf.dist</li>
    <li>python312Packages.pytikz-allefeld</li>
    <li>python312Packages.pytikz-allefeld.dist</li>
    <li>qalculate-gtk</li>
    <li>remmina</li>
    <li>smuview</li>
    <li>spatialite_gui</li>
    <li>supersonic</li>
    <li>termpdfpy</li>
    <li>termpdfpy.dist</li>
    <li>texpresso</li>
    <li>thonny</li>
    <li>thonny.dist</li>
    <li>viking</li>
    <li>vimPlugins.openscad-nvim</li>
    <li>virt-manager</li>
    <li>virt-manager.dist</li>
    <li>zathura</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 25 packages marked as broken and skipped:</summary>
  <ul>
    <li>gimpPlugins.exposureBlend</li>
    <li>gimpPlugins.resynthesizer</li>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>private-gpt</li>
    <li>private-gpt.dist</li>
    <li>python311Packages.llama-index</li>
    <li>python311Packages.llama-index-readers-file</li>
    <li>python311Packages.llama-index-readers-file.dist</li>
    <li>python311Packages.llama-index-readers-s3</li>
    <li>python311Packages.llama-index-readers-s3.dist</li>
    <li>python311Packages.llama-index.dist</li>
    <li>python311Packages.private-gpt</li>
    <li>python311Packages.private-gpt.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-readers-file</li>
    <li>python312Packages.llama-index-readers-file.dist</li>
    <li>python312Packages.llama-index-readers-s3</li>
    <li>python312Packages.llama-index-readers-s3.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.private-gpt</li>
    <li>python312Packages.private-gpt.dist</li>
    <li>rtfm</li>
    <li>tijolo</li>
    <li>volantes-cursors</li>
  </ul>
</details>
<details>
  <summary>:x: 8 packages failed to build:</summary>
  <ul>
    <li>emojione</li>
    <li>gimpPlugins.bimp</li>
    <li>gimpPlugins.gap</li>
    <li>hackneyed</li>
    <li>python311Packages.pdf2docx</li>
    <li>python311Packages.pdf2docx.dist</li>
    <li>saga</li>
    <li>sioyek</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 105 packages built:</summary>
  <ul>
    <li>apkleaks</li>
    <li>apkleaks.dist</li>
    <li>asciidoc-full</li>
    <li>asciidoc-full-with-plugins</li>
    <li>asciidoc-full-with-plugins.dist</li>
    <li>asciidoc-full.dist</li>
    <li>azpainter</li>
    <li>beamerpresenter</li>
    <li>browsr</li>
    <li>browsr.dist</li>
    <li>cadzinho</li>
    <li>dblatexFull</li>
    <li>desktopToDarwinBundle</li>
    <li>diagrams-as-code</li>
    <li>diagrams-as-code.dist</li>
    <li>dsview</li>
    <li>dune3d</li>
    <li>element-desktop</li>
    <li>element-desktop-wayland</li>
    <li>fastqc</li>
    <li>gcs</li>
    <li>ghidra</li>
    <li>ghidra-extensions.findcrypt</li>
    <li>ghidra-extensions.ghidra-delinker-extension</li>
    <li>ghidra-extensions.ghidra-golanganalyzerextension</li>
    <li>ghidra-extensions.gnudisassembler</li>
    <li>ghidra-extensions.lightkeeper</li>
    <li>ghidra-extensions.machinelearning</li>
    <li>ghidra-extensions.ret-sync</li>
    <li>ghidra-extensions.sleighdevtools</li>
    <li>gimp</li>
    <li>gimp.dev</li>
    <li>gimpPlugins.farbfeld</li>
    <li>gimpPlugins.fourier</li>
    <li>gimpPlugins.gmic</li>
    <li>gimpPlugins.lightning</li>
    <li>gimpPlugins.lqrPlugin</li>
    <li>gimpPlugins.texturize</li>
    <li>gimpPlugins.waveletSharpen</li>
    <li>gpscorrelate</li>
    <li>gtkwave</li>
    <li>hikounomizu</li>
    <li>inkscape</li>
    <li>inkscape-extensions.textext</li>
    <li>inkscape-extensions.textext.dist</li>
    <li>inkscape-with-extensions</li>
    <li>jadx</li>
    <li>k40-whisperer</li>
    <li>kakounePlugins.rep</li>
    <li>librum</li>
    <li>llpp</li>
    <li>logisim</li>
    <li>logisim-evolution</li>
    <li>lyra-cursors</li>
    <li>manga-cli</li>
    <li>mars-mips</li>
    <li>mcomix</li>
    <li>mcomix.dist</li>
    <li>meld</li>
    <li>mupdf</li>
    <li>mupdf.bin</li>
    <li>mupdf.dev</li>
    <li>mupdf.doc</li>
    <li>mupdf.man</li>
    <li>numix-cursor-theme</li>
    <li>opcua-client-gui</li>
    <li>opcua-client-gui.dist</li>
    <li>pulseview</li>
    <li>python311Packages.diagrams</li>
    <li>python311Packages.diagrams.dist</li>
    <li>python311Packages.icnsutil</li>
    <li>python311Packages.icnsutil.dist</li>
    <li>python311Packages.osc-diagram</li>
    <li>python311Packages.osc-diagram.dist</li>
    <li>python311Packages.pymupdf</li>
    <li>python311Packages.pymupdf.dist</li>
    <li>python311Packages.pytikz-allefeld</li>
    <li>python311Packages.pytikz-allefeld.dist</li>
    <li>python312Packages.diagrams</li>
    <li>python312Packages.diagrams.dist</li>
    <li>python312Packages.icnsutil</li>
    <li>python312Packages.icnsutil.dist</li>
    <li>python312Packages.osc-diagram</li>
    <li>python312Packages.osc-diagram.dist</li>
    <li>python312Packages.pdf2docx</li>
    <li>python312Packages.pdf2docx.dist</li>
    <li>python312Packages.pymupdf</li>
    <li>python312Packages.pymupdf.dist</li>
    <li>python312Packages.pytikz-allefeld</li>
    <li>python312Packages.pytikz-allefeld.dist</li>
    <li>qalculate-gtk</li>
    <li>remmina</li>
    <li>smuview</li>
    <li>spatialite_gui</li>
    <li>supersonic</li>
    <li>termpdfpy</li>
    <li>termpdfpy.dist</li>
    <li>texpresso</li>
    <li>thonny</li>
    <li>thonny.dist</li>
    <li>viking</li>
    <li>vimPlugins.openscad-nvim</li>
    <li>virt-manager</li>
    <li>virt-manager.dist</li>
    <li>zathura</li>
  </ul>
</details>

**Comment by carlossless** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 358247`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 5 packages built:</summary>
  <ul>
    <li>desktopToDarwinBundle</li>
    <li>python311Packages.icnsutil</li>
    <li>python311Packages.icnsutil.dist</li>
    <li>python312Packages.icnsutil</li>
    <li>python312Packages.icnsutil.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 5 packages built:</summary>
  <ul>
    <li>desktopToDarwinBundle</li>
    <li>python311Packages.icnsutil</li>
    <li>python311Packages.icnsutil.dist</li>
    <li>python312Packages.icnsutil</li>
    <li>python312Packages.icnsutil.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:fast_forward: 21 packages marked as broken and skipped:</summary>
  <ul>
    <li>ghidra</li>
    <li>ghidra-extensions.findcrypt</li>
    <li>ghidra-extensions.ghidra-delinker-extension</li>
    <li>ghidra-extensions.ghidra-golanganalyzerextension</li>
    <li>ghidra-extensions.gnudisassembler</li>
    <li>ghidra-extensions.lightkeeper</li>
    <li>ghidra-extensions.machinelearning</li>
    <li>ghidra-extensions.ret-sync</li>
    <li>ghidra-extensions.sleighdevtools</li>
    <li>gimpPlugins.exposureBlend</li>
    <li>gimpPlugins.resynthesizer</li>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>private-gpt</li>
    <li>private-gpt.dist</li>
    <li>python311Packages.private-gpt</li>
    <li>python311Packages.private-gpt.dist</li>
    <li>python312Packages.private-gpt</li>
    <li>python312Packages.private-gpt.dist</li>
    <li>rtfm</li>
    <li>tijolo</li>
  </ul>
</details>
<details>
  <summary>:x: 12 packages failed to build:</summary>
  <ul>
    <li>dune3d</li>
    <li>emojione</li>
    <li>gimpPlugins.bimp</li>
    <li>gimpPlugins.gap</li>
    <li>librum</li>
    <li>opcua-client-gui</li>
    <li>opcua-client-gui.dist</li>
    <li>python311Packages.pdf2docx</li>
    <li>python311Packages.pdf2docx.dist</li>
    <li>saga</li>
    <li>sioyek</li>
    <li>volantes-cursors</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 106 packages built:</summary>
  <ul>
    <li>apkleaks</li>
    <li>apkleaks.dist</li>
    <li>asciidoc-full</li>
    <li>asciidoc-full-with-plugins</li>
    <li>asciidoc-full-with-plugins.dist</li>
    <li>asciidoc-full.dist</li>
    <li>azpainter</li>
    <li>beamerpresenter</li>
    <li>browsr</li>
    <li>browsr.dist</li>
    <li>cadzinho</li>
    <li>dblatexFull</li>
    <li>ddd</li>
    <li>desktopToDarwinBundle</li>
    <li>diagrams-as-code</li>
    <li>diagrams-as-code.dist</li>
    <li>dsview</li>
    <li>element-desktop</li>
    <li>element-desktop-wayland</li>
    <li>fastqc</li>
    <li>gcs</li>
    <li>gimp</li>
    <li>gimp.dev</li>
    <li>gimpPlugins.farbfeld</li>
    <li>gimpPlugins.fourier</li>
    <li>gimpPlugins.gmic</li>
    <li>gimpPlugins.lightning</li>
    <li>gimpPlugins.lqrPlugin</li>
    <li>gimpPlugins.texturize</li>
    <li>gimpPlugins.waveletSharpen</li>
    <li>gpscorrelate</li>
    <li>gtkwave</li>
    <li>hackneyed</li>
    <li>hikounomizu</li>
    <li>inkscape</li>
    <li>inkscape-extensions.textext</li>
    <li>inkscape-extensions.textext.dist</li>
    <li>inkscape-with-extensions</li>
    <li>jadx</li>
    <li>k40-whisperer</li>
    <li>kakounePlugins.rep</li>
    <li>llpp</li>
    <li>logisim</li>
    <li>logisim-evolution</li>
    <li>lyra-cursors</li>
    <li>manga-cli</li>
    <li>mars-mips</li>
    <li>mcomix</li>
    <li>mcomix.dist</li>
    <li>meld</li>
    <li>mupdf</li>
    <li>mupdf.bin</li>
    <li>mupdf.dev</li>
    <li>mupdf.doc</li>
    <li>mupdf.man</li>
    <li>numix-cursor-theme</li>
    <li>pulseview</li>
    <li>python311Packages.diagrams</li>
    <li>python311Packages.diagrams.dist</li>
    <li>python311Packages.icnsutil</li>
    <li>python311Packages.icnsutil.dist</li>
    <li>python311Packages.llama-index</li>
    <li>python311Packages.llama-index-readers-file</li>
    <li>python311Packages.llama-index-readers-file.dist</li>
    <li>python311Packages.llama-index-readers-s3</li>
    <li>python311Packages.llama-index-readers-s3.dist</li>
    <li>python311Packages.llama-index.dist</li>
    <li>python311Packages.osc-diagram</li>
    <li>python311Packages.osc-diagram.dist</li>
    <li>python311Packages.pymupdf</li>
    <li>python311Packages.pymupdf.dist</li>
    <li>python311Packages.pytikz-allefeld</li>
    <li>python311Packages.pytikz-allefeld.dist</li>
    <li>python312Packages.diagrams</li>
    <li>python312Packages.diagrams.dist</li>
    <li>python312Packages.icnsutil</li>
    <li>python312Packages.icnsutil.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-readers-file</li>
    <li>python312Packages.llama-index-readers-file.dist</li>
    <li>python312Packages.llama-index-readers-s3</li>
    <li>python312Packages.llama-index-readers-s3.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.osc-diagram</li>
    <li>python312Packages.osc-diagram.dist</li>
    <li>python312Packages.pdf2docx</li>
    <li>python312Packages.pdf2docx.dist</li>
    <li>python312Packages.pymupdf</li>
    <li>python312Packages.pymupdf.dist</li>
    <li>python312Packages.pytikz-allefeld</li>
    <li>python312Packages.pytikz-allefeld.dist</li>
    <li>qalculate-gtk</li>
    <li>remmina</li>
    <li>smuview</li>
    <li>spatialite_gui</li>
    <li>supersonic</li>
    <li>termpdfpy</li>
    <li>termpdfpy.dist</li>
    <li>texpresso</li>
    <li>thonny</li>
    <li>thonny.dist</li>
    <li>viking</li>
    <li>vimPlugins.openscad-nvim</li>
    <li>virt-manager</li>
    <li>virt-manager.dist</li>
    <li>zathura</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 25 packages marked as broken and skipped:</summary>
  <ul>
    <li>gimpPlugins.exposureBlend</li>
    <li>gimpPlugins.resynthesizer</li>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>private-gpt</li>
    <li>private-gpt.dist</li>
    <li>python311Packages.llama-index</li>
    <li>python311Packages.llama-index-readers-file</li>
    <li>python311Packages.llama-index-readers-file.dist</li>
    <li>python311Packages.llama-index-readers-s3</li>
    <li>python311Packages.llama-index-readers-s3.dist</li>
    <li>python311Packages.llama-index.dist</li>
    <li>python311Packages.private-gpt</li>
    <li>python311Packages.private-gpt.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-readers-file</li>
    <li>python312Packages.llama-index-readers-file.dist</li>
    <li>python312Packages.llama-index-readers-s3</li>
    <li>python312Packages.llama-index-readers-s3.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.private-gpt</li>
    <li>python312Packages.private-gpt.dist</li>
    <li>rtfm</li>
    <li>tijolo</li>
    <li>volantes-cursors</li>
  </ul>
</details>
<details>
  <summary>:x: 7 packages failed to build:</summary>
  <ul>
    <li>emojione</li>
    <li>gimpPlugins.bimp</li>
    <li>gimpPlugins.gap</li>
    <li>python311Packages.pdf2docx</li>
    <li>python311Packages.pdf2docx.dist</li>
    <li>saga</li>
    <li>sioyek</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 106 packages built:</summary>
  <ul>
    <li>apkleaks</li>
    <li>apkleaks.dist</li>
    <li>asciidoc-full</li>
    <li>asciidoc-full-with-plugins</li>
    <li>asciidoc-full-with-plugins.dist</li>
    <li>asciidoc-full.dist</li>
    <li>azpainter</li>
    <li>beamerpresenter</li>
    <li>browsr</li>
    <li>browsr.dist</li>
    <li>cadzinho</li>
    <li>dblatexFull</li>
    <li>desktopToDarwinBundle</li>
    <li>diagrams-as-code</li>
    <li>diagrams-as-code.dist</li>
    <li>dsview</li>
    <li>dune3d</li>
    <li>element-desktop</li>
    <li>element-desktop-wayland</li>
    <li>fastqc</li>
    <li>gcs</li>
    <li>ghidra</li>
    <li>ghidra-extensions.findcrypt</li>
    <li>ghidra-extensions.ghidra-delinker-extension</li>
    <li>ghidra-extensions.ghidra-golanganalyzerextension</li>
    <li>ghidra-extensions.gnudisassembler</li>
    <li>ghidra-extensions.lightkeeper</li>
    <li>ghidra-extensions.machinelearning</li>
    <li>ghidra-extensions.ret-sync</li>
    <li>ghidra-extensions.sleighdevtools</li>
    <li>gimp</li>
    <li>gimp.dev</li>
    <li>gimpPlugins.farbfeld</li>
    <li>gimpPlugins.fourier</li>
    <li>gimpPlugins.gmic</li>
    <li>gimpPlugins.lightning</li>
    <li>gimpPlugins.lqrPlugin</li>
    <li>gimpPlugins.texturize</li>
    <li>gimpPlugins.waveletSharpen</li>
    <li>gpscorrelate</li>
    <li>gtkwave</li>
    <li>hackneyed</li>
    <li>hikounomizu</li>
    <li>inkscape</li>
    <li>inkscape-extensions.textext</li>
    <li>inkscape-extensions.textext.dist</li>
    <li>inkscape-with-extensions</li>
    <li>jadx</li>
    <li>k40-whisperer</li>
    <li>kakounePlugins.rep</li>
    <li>librum</li>
    <li>llpp</li>
    <li>logisim</li>
    <li>logisim-evolution</li>
    <li>lyra-cursors</li>
    <li>manga-cli</li>
    <li>mars-mips</li>
    <li>mcomix</li>
    <li>mcomix.dist</li>
    <li>meld</li>
    <li>mupdf</li>
    <li>mupdf.bin</li>
    <li>mupdf.dev</li>
    <li>mupdf.doc</li>
    <li>mupdf.man</li>
    <li>numix-cursor-theme</li>
    <li>opcua-client-gui</li>
    <li>opcua-client-gui.dist</li>
    <li>pulseview</li>
    <li>python311Packages.diagrams</li>
    <li>python311Packages.diagrams.dist</li>
    <li>python311Packages.icnsutil</li>
    <li>python311Packages.icnsutil.dist</li>
    <li>python311Packages.osc-diagram</li>
    <li>python311Packages.osc-diagram.dist</li>
    <li>python311Packages.pymupdf</li>
    <li>python311Packages.pymupdf.dist</li>
    <li>python311Packages.pytikz-allefeld</li>
    <li>python311Packages.pytikz-allefeld.dist</li>
    <li>python312Packages.diagrams</li>
    <li>python312Packages.diagrams.dist</li>
    <li>python312Packages.icnsutil</li>
    <li>python312Packages.icnsutil.dist</li>
    <li>python312Packages.osc-diagram</li>
    <li>python312Packages.osc-diagram.dist</li>
    <li>python312Packages.pdf2docx</li>
    <li>python312Packages.pdf2docx.dist</li>
    <li>python312Packages.pymupdf</li>
    <li>python312Packages.pymupdf.dist</li>
    <li>python312Packages.pytikz-allefeld</li>
    <li>python312Packages.pytikz-allefeld.dist</li>
    <li>qalculate-gtk</li>
    <li>remmina</li>
    <li>smuview</li>
    <li>spatialite_gui</li>
    <li>supersonic</li>
    <li>termpdfpy</li>
    <li>termpdfpy.dist</li>
    <li>texpresso</li>
    <li>thonny</li>
    <li>thonny.dist</li>
    <li>viking</li>
    <li>vimPlugins.openscad-nvim</li>
    <li>virt-manager</li>
    <li>virt-manager.dist</li>
    <li>zathura</li>
  </ul>
</details>

**Comment by carlossless** [comment]
> nixpkgs-review result

build failures seem unrelated to changes in this script

File: pkgs/build-support/setup-hooks/desktop-to-darwin-bundle.sh
Comment (emilazy) [inline pkgs/build-support/setup-hooks/desktop-to-darwin-bundle.sh:118]: We’re about to switch unstable to require macOS ≥ 11.3; what would need changing here to use ARGB? (We can keep it like this in the current PR for backport to 24.11).

**Review by emilazy — COMMENTED** [review]
LGTM in principle.

File: pkgs/build-support/setup-hooks/desktop-to-darwin-bundle.sh
Comment (carlossless) [reply pkgs/build-support/setup-hooks/desktop-to-darwin-bundle.sh:118]: It would only be a matter of:
```
diff --git a/pkgs/build-support/setup-hooks/desktop-to-darwin-bundle.sh b/pkgs/build-support/setup-hooks/desktop-to-darwin-bundle.sh
index 4fbef4baa512..6790754f740a 100644
--- a/pkgs/build-support/setup-hooks/desktop-to-darwin-bundle.sh
+++ b/pkgs/build-support/setup-hooks/desktop-to-darwin-bundle.sh
@@ -115,15 +115,15 @@ convertIconTheme() {
     }
 
     # macOS does not correctly display 16x and 32x png icons on app bundles
-    # they need to be converted to rgb+mask (argb is supported only from macOS 11)
+    # they need to be converted to argb
     function convertIfUnsupportedIcon() {
         local -r in=$1
         local -r iconSize=$2
         local -r scale=$3
-        local -r out=${in%.png}.rgb
+        local -r out=${in%.png}.argb
 
         if [[ ($scale -eq 1) && ($iconSize -eq 32 || $iconSize -eq 16) ]]; then
-            echo "desktopToDarwinBundle: converting ${iconSize}x icon to rgb" >&2
+            echo "desktopToDarwinBundle: converting ${iconSize}x icon to argb" >&2
             icnsutil convert "$out" "$in"
             rm "$in"
         fi
```

This would be the result:
relikd github io_icnsutil_html_viewer html

**Review by emilazy — APPROVED** [review]
Thanks! Was waiting on CI before merge but it looks good to me. I’ll handle the ARGB switch on `staging`.

**Pull request merged**
